src/Tribe/Views/V2/Views/By_Day_View.php

```diff
@@ -486,15 +488,39 @@ protected function backfill_multiday_event_ids( array $events = [] ) {
 				continue;
 			}
 
+			$site_timezone     = Timezones::build_timezone_object();
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: d46a7924dab829ebf8caad6f68ceaf2f</sub>